root/user/UserTagList.js

```diff
@@ -61,84 +97,113 @@ const UserTagList = ({
   sortBy,
   tags,
   user,
-}: Props): React.Element<typeof UserAccountLayout> => (
-  <UserAccountLayout entity={user} page="tags" title={l('Tags')}>
-    <h2>
-      {getTagListHeading(user.name, showDownvoted)}
-    </h2>
-
-    <UserTagFilters
-      $c={$c}
-      showDownvoted={showDownvoted}
-      showSortSelect
-      showVotesSelect
-      sortBy={sortBy}
-    />
-
-    <div id="all-tags">
-      {(genres.length > 0 || tags.length > 0) ? (
-        <>
-          <h3>{l('Genres')}</h3>
-
-          <div id="genres">
-            {genres.length > 0 ? (
-              <ul className="genre-list">
-                {genres.map((tag, index) => (
-                  <li className={loopParity(index)} key={tag.tag.id}>
-                    <UserTagLink
-                      showDownvoted={showDownvoted}
-                      tag={tag.tag.name}
-                      username={user.name}
-                    />
-                    <span className="tag-vote-buttons">
-                      <span className="tag-count">{tag.count}</span>
-                    </span>
-                  </li>
-                ))}
-              </ul>
-            ) : <p>{l('There are no genres to show.')}</p>}
-          </div>
-
-          <h3>{l('Other tags')}</h3>
-
-          <div id="tags">
-            {tags.length > 0 ? (
-              <ul className="tag-list">
-                {tags.map((tag, index) => (
-                  <li className={loopParity(index)} key={tag.tag.id}>
-                    <UserTagLink
-                      showDownvoted={showDownvoted}
-                      tag={tag.tag.name}
-                      username={user.name}
-                    />
-                    <span className="tag-vote-buttons">
-                      <span className="tag-count">{tag.count}</span>
-                    </span>
-                  </li>
-                ))}
-              </ul>
-            ) : <p>{l('There are no other tags to show.')}</p>}
-          </div>
-        </>
-      ) : (
-        showDownvoted ? (
-          <p>
-            {exp.l(
-              '{user} has not downvoted any tags.',
-              {user: <EditorLink editor={user} />},
-            )}
-          </p>
+}: Props): React.Element<typeof UserAccountLayout> => {
+  const viewingOwnTags = Boolean($c.user && user &&
+                                 $c.user.id === user.id);
+
+  return (
+    <UserAccountLayout entity={user} page="tags" title={l('Tags')}>
+      <h2>
+        {getTagListHeading(user.name, showDownvoted)}
+      </h2>
+
+      <UserTagFilters
+        $c={$c}
+        showDownvoted={showDownvoted}
+        showSortSelect
+        showVotesSelect
+        sortBy={sortBy}
+      />
+
+      <div id="all-tags">
+        {(genres.length > 0 || tags.length > 0) ? (
+          <>
+            <h3>{l('Genres')}</h3>
+
+            <div id="genres">
+              {genres.length > 0 ? (
```

Comment:
I split the tag "pop rap" into "pop" and "rap" and for some reason pop is now displayed twice under genres.  Is that a new issue?

Reply:
I cannot reproduce that myself, weird. Can you?

Reply:
Can't reproduce.